apps/tunnel/unabto_config.h

```diff
@@ -28,6 +28,8 @@
 #define NABTO_ENABLE_TCP_FALLBACK 1
 
 #define NABTO_ENABLE_TUNNEL_TCP 1
+#define NABTO_ENABLE_TUNNEL_UART 1
```

Comment:
Are these neccessary when +set(ENABLE_TUNNEL_UART 1) is added to the cmake file?